ci/moose_utils.py

```diff
@@ -43,6 +43,8 @@ def _download_moose_file(opsys: str, arch: str, file_name: str):
         raise ValueError("LIBTELIO_ENV_SEC_NEXUS_URL not set")
 
     url = f"{nexus_url}/repository/ll-gitlab-release/{MOOSE_PROJECT_ID}/{LIBTELIO_ENV_MOOSE_RELEASE_TAG}/bin/common/{opsys}/{arch}/{file_name}"
+    if not url.startswith("https://"):
+        url = "https://" + url
```

Comment:
This may provide invalid URL if it is prefixed with `http://` or similar other protocol types. Not sure how relevant this is though :thinking:.

Reply:
This is not necessary, I needed to edit my local env vars, probably other users will also need to.

Reply:
ok